Stratis.Bitcoin/Connection/ConnectionManagerBehavior.cs

```diff
@@ -54,29 +40,31 @@ protected override void AttachCore()
 			this.chainHeadersBehavior = this.AttachedNode.Behaviors.Find<ChainHeadersBehavior>();
 		}
 
-		ChainHeadersBehavior chainHeadersBehavior;
 		private void AttachedNode_MessageReceived(Node node, IncomingMessage message)
 		{
-			if(this.chainHeadersBehavior.InvalidHeaderReceived && !this.Whitelisted)
+			if (this.chainHeadersBehavior.InvalidHeaderReceived && !this.Whitelisted)
 			{
 				node.DisconnectAsync("Invalid block received");
 			}
 		}
 
 		private void AttachedNode_StateChanged(Node node, NodeState oldState)
 		{
-			if(node.State == NodeState.HandShaked)
+			if (node.State == NodeState.HandShaked)
 			{
 				this.ConnectionManager.AddConnectedNode(node);
-				this.logger.LogInformation("Node " + node.RemoteSocketEndpoint + " connected (" + (this.Inbound ? "inbound" : "outbound") + "), agent " + node.PeerVersion.UserAgent + ", height " + node.PeerVersion.StartHeight);
+				this.logger.LogInformation("Node {0} connected ({1}), agent {2}, height {3}", node.RemoteSocketEndpoint, this.Inbound ? "inbound" : "outbound", node.PeerVersion.UserAgent, node.PeerVersion.StartHeight);
 				node.SendMessageAsync(new SendHeadersPayload());
 			}
-			if(node.State == NodeState.Failed || node.State == NodeState.Offline)
+
+			if ((node.State == NodeState.Failed) || (node.State == NodeState.Offline))
 			{
-			    this.logger.LogInformation("Node " + node.RemoteSocketEndpoint + " offline");
-				if(node.DisconnectReason != null && !string.IsNullOrEmpty(node.DisconnectReason.Reason))
-				    this.logger.LogInformation("Reason: " + node.DisconnectReason.Reason);
-				this.ConnectionManager.RemoveConnectedNode(node);
+			    this.logger.LogInformation("Node {0} offline.", node.RemoteSocketEndpoint);
+
+                if (node.DisconnectReason != null && !string.IsNullOrEmpty(node.DisconnectReason.Reason))
+				    this.logger.LogInformation("Reason: {0}", node.DisconnectReason.Reason);
+
+                this.ConnectionManager.RemoveConnectedNode(node);
```

Comment:
tabs vs spaces formatting

Reply:
i will soon probably create PR to untabify whole solution

Reply:
but fixed here